src/components/app_header/header/HomePageHeader.tsx

```diff
@@ -41,7 +41,7 @@ const HomepageHeader = () => {
             <br />
             <button
               type="button"
-              className="font-bold bg-titansBrightPink text-white font-bold py-2 px-8 rounded-full"
+              className="bg-titansBrightPink text-white font-bold py-2 px-8 rounded-full"
```

Comment:
why change `font-bold`?

Reply:
Ah I see its just the order of classes that has changed

Reply:
font-bold was in there twice for some reason

Reply:
Logo was just the shade of pink changing in the SVG - Zoya will update on figma also